workflows/cloudify_system_workflows/snapshots/postgres.py

```diff
@@ -336,8 +337,13 @@ def _restore_dump(self, dump_file, db_name, table=None):
         run_shell(command)
 
     @staticmethod
-    def _append_dump(dump_file, query):
-        ctx.logger.debug('Adding to end of dump: {0}'.format(query))
+    def _append_dump(dump_file, query, without_password=True):
+        if without_password:
+            ctx.logger.debug('Adding to end of dump: {0}'.format(query))
+        else:
+            ctx.logger.debug("Adding to end of dump: UPDATE users SET "
```

Comment:
You have duplicate code here, and [here](https://github.com/cloudify-cosmo/cloudify-manager/blob/master/workflows/cloudify_system_workflows/snapshots/postgres.py#L180)
I would update the `_get_admin_user_update_query` to receive `username` and `password` as `None` and in case not None put the strings received instead of getting the credentials from `_get_admin_credentials`.
That way also the string will stay consistent in case it changes in the future.

Reply:
I'm not quite sure I understood what you meant. When `append_dump` receives a query that contains credentials it should print the query with the credentials hidden by asterisk but append the real query to the dump file, how this change helps us with it? 
In any case, your point is right, we should make the function more generic. I'll work on it.